go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -188,10 +188,24 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb string, sourceIP string, s
 	vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 	// NOTE: doesn't use vip, to avoid having brackets in the name with IPv6
 	aclName := fmt.Sprintf("%s-%s:%d", lb, sourceIP, sourcePort)
+	aclName = strings.ReplaceAll(aclName, ":", "\\:")
```

Comment:
can we do this a function so we can add unit tests later?
```go
func generateACLName(lb, sourceIP, sourcePort string) string {
}
```

we can follow up in the unit test later if you are too busy, just add a FIXME or TODO comment o not forget

Reply:
Moved to function but forgot the unit test FIXME comment.

Reply:
Added comment. Now done.